libsupport/include/katana/EntityTypeManager.h

```diff
@@ -137,12 +137,6 @@ class KATANA_EXPORT EntityTypeManager {
     // assign the type ID for each row
     for (int64_t row = 0; row < num_rows; ++row) {
       TypeProperties::FieldEntity field_indices;
-      for (auto& bool_property : type_properties.bool_properties) {
```

Comment:
So it's only uint8 properties now?

Reply:
Yes. @roshandathathri says we don't need to support boolean properties any more.